### PR TITLE
Faster Protobufs

### PR DESCRIPTION
**Pull request by rockwood**
This extracts a RealtimeProtobuf class that's responsible for fetching and caching realtime updates.

### CODE REVIEW

**Comment by rockwood** [comment]
@geofflane could you take a gander at this

File: app/apis/metro/realtime_updates.rb [hunk 1]
```diff
@@ -3,27 +3,15 @@
```
Comment (geofflane) [inline app/apis/metro/realtime_updates.rb]: Do we need the 'gtfs-realtime.pb.rb' require still?

**Comment by geofflane** [comment]
Overall looks good. Is it faster? Or do you need to wait and instrument it to see?

**Comment by rockwood** [comment]
It'll definitely be faster since we don't have to parse the protobuf on each request. The next step is to move the fetch into a job and have each request read from the cache.

**Pull request merged**